build/buildspec.docker.dev.yml

```diff
@@ -17,24 +17,31 @@ phases:
       - chmod 600 ~/.ssh/ssh_key
       - eval "$(ssh-agent -s)"
       - ssh-add ~/.ssh/ssh_key
+      - go get github.com/onsi/ginkgo/ginkgo
   build:
     commands:
       - export GO111MODULE=on
       - DATE=$(date "+%F,%T")
       - VERSION=$(echo $GIT_BRANCH | tr "/" .)
       # Build Open source version
+      - ginkgo --randomizeAllSpecs --randomizeSuites --failOnPending --cover -coverprofile=profile.out -covermode=atomic --trace --progress -p --skipPackage=tests ./...
```

Comment:
can you add 
go tool cover -html=profile.out -o coverage.html and dump it to s3?